From abcb9eeacc2f12c3d77b66c274ace9bffecf68e3 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Tue, 23 Jul 2019 02:48:43 +0200
Subject: [PATCH libaio 20/28] harness: Make the test exit with a code matching
 the pass/fail state

This way we can use the exit code to check whether the tests passed or
failed, and fail the package build.

Signed-off-by: Guillem Jover <guillem@hadrons.org>
Signed-off-by: Jeff Moyer <jmoyer@redhat.com>
---
 harness/runtests.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/harness/runtests.sh b/harness/runtests.sh
index 717c72a..ef269a7 100755
--- a/harness/runtests.sh
+++ b/harness/runtests.sh
@@ -17,3 +17,5 @@ done
 
 echo "Pass: $passes  Fail: $fails"
 echo "Test run complete at" `date`
+
+exit $fails
-- 
2.26.0.292.g33ef6b2f38

